packages/mobile/src/styles/makeStyles.ts

```diff
@@ -61,7 +64,14 @@ export const makeStyles = <PropsT, T extends NamedStyles<T> = NamedStyles<any>>(
     const stylesheet = useMemo(() => {
       const theme = { palette, typography, spacing, type: themeVariant }
       const namedStyles = styles(theme, memoizedProps)
-      return StyleSheet.create(namedStyles)
+      const namedStylesHash = hash(namedStyles)
```

Comment:
this is cool. 

is it worth us thinking about moving styles out of the render cycle? (not sure how we would do theme changes but maybe possible)

Reply:
yeah i would love to do that, but how do we handle theming? there doesn't seem to be a good pattern online for using Stylesheet directly like this, so might be worth considering emotionjs or some other css-in-jss-native option. long term question though imo.

Reply:
yes 100%